matrix/plugins/glitch/utils.py

```diff
@@ -0,0 +1,17 @@
+from juju.client.client import ClientFacade
+import subprocess
+import yaml
+
+async def is_leader(model, rule, unit_name):
```

Comment:
@johnsca PR for moving this into python-libjuju here: https://github.com/juju/python-libjuju/pull/13

(I dealt with my concerns about hackiness and conflicting with a non asynchronous call in the future by just giving it a slightly scary name that won't conflict with the name of the "right" method in the future ...)